tools/verdict/com.ge.research.osate.verdict/src/com/ge/research/osate/verdict/aadl2vdm/Aadl2Vdm.java

```diff
@@ -217,57 +271,79 @@ public Model populateVDMFromAadlObjects(List<EObject> objects, Model model) {
 
 						switch(propCat) {
 							case "system": {
-								componentPropertyToName.put(prop, propName);
+								if(objectNamesFromFilesInProject.contains(propName)) {
+									componentPropertyToName.put(prop, propName);
+								}
```

Comment:
I had the same thought until I noticed that the content inside the if branch is not the same for all cases, "componentPropertyToName" is getting updated in some cases and "connPropertyToName" is getting updated in case of a connection.